metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -296,15 +346,29 @@ private static CreatePublicationRequest requestWithLanguage(URI language) {
 
     private static Arguments generateTestHtml(String testDescription, String name, String content,
                                               CreatePublicationRequest expected) {
-        return Arguments.of(testDescription, html(head(meta().withName(name).withContent(content))).renderFormatted(),
+        return Arguments.of(testDescription, generateHtmlWithSingleMetaTag(name, content),
```

Comment:
Surely we don't need _two_ single line helper methods like this. If the first one was not clear enough I suggest changing the name of it rather than moving the logic to another method.

Reply:
I should have been clear, this is a step in the refactoring. The old code will be removed entirely in a subsequent PR

Reply:
Ah, I see. Never mind then :)